fix: adds ReturnTypeWillChange to temporarily ignore deprecation warnings

Justintime50 opened the pull request:
Adds `#[\ReturnTypeWillChange]` to some EasyPostObject methods to remove the deprecation warnings on PHP 8.1. The return type definitions will be required for PHP 9; however, because we will still be supporting PHP 7.3 for some time and the `mixed` return type which is required to fix this but not included until PHP 8 isn't possible at the moment, we'll temporarily ignore these deprecation notices until we drop support for PHP 7 altogether (I'd assume about the same time we adopt PHP 9 once it's out.)

Closes #124

jchen293 commented:
I have PHP 8.1.1 and did some testing on this, it looks like **offsetGet()** and **key()** are also throwing deprecation warnings.

Justintime50 commented:
Great catch @jchen293! Must have been removed when I switched from return types to ignoring them for now. I've added the ignore comments to those two as well.
